Sources/Quick/DSL/World+DSL.swift

```diff
@@ -20,8 +20,13 @@ extension World {
 
     internal func describe(description: String, flags: FilterFlags, closure: () -> ()) {
         guard currentExampleGroup != nil else {
-            NSException(name: NSInternalInconsistencyException, reason:"Error: example group was not created by its parent QuickSpec spec. Check that describe() or context() was used in QuickSpec.spec() and not a more general context (i.e. an XCTestCase test)", userInfo: nil).raise()
-            return
+            let error = "Error: example group was not created by its parent QuickSpec spec. Check that describe() or context() was used in QuickSpec.spec() and not a more general context (i.e. an XCTestCase test)"
+            #if os(Linux)
+                fatalError(error)
+            #else
+                NSException(name: NSInternalInconsistencyException, reason: error, userInfo: nil).raise()
+                return ()
+            #endif
```

Comment:
Testing precondition/assert/fatalError has been [brought up in the swift corelibs mailing list](https://lists.swift.org/pipermail/swift-corelibs-dev/Week-of-Mon-20151207/000166.html), but I'm not sure if there's any work being done on it.